### PR TITLE
Format the python server code

### PR DESCRIPTION
**Pull request by yijie-yang**

### CODE REVIEW

File: example_colabs/custom_data_overlay_demo.ipynb [hunk 1]
```diff
@@ -1,310 +1,317 @@
...
-      "provenance": []
-    },
-    "kernelspec": {
-      "display_name": "Python 3",
-      "name": "python3"
-    },
-    "language_info": {
-      "name": "python"
-    }
+ "cells": [
```
Comment (jinjingforever) [inline example_colabs/custom_data_overlay_demo.ipynb:2]: Just curious, what are the changes in these two notebooks? (hard to tell from github diff)

File: format.sh [hunk 2]
```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
```
Comment (jinjingforever) [inline format.sh:1]: Would this script and the format changes necessary if we will eventually move the sot to internal codebase where the format will be different?

File: example_colabs/custom_data_overlay_demo.ipynb (hunk 1 above)
Comment (yijie-yang) [reply example_colabs/custom_data_overlay_demo.ipynb:2]: It did some indentation formatting of the colab scripts (changing from 4 spaces to 2 spaces).

File: format.sh (hunk 2 above)
Comment (yijie-yang) [reply format.sh:1]: This helps the external contributors to do the formatting before copying the code to internal for review. The format rules should be the same as internal codebase.

Comment (jinjingforever) [reply format.sh:1]: oh nice! good to know. thanks!

**Review by jinjingforever — APPROVED** [review]

**Comment by yijie-yang** [comment]
Let me fix all the license header issues before merging this PR. After all issues fixed, the "Run Server Code CI / Formatting" should turn green.

**Pull request merged**